homeassistant/components/notify/clicksend.py

```diff
@@ -0,0 +1,71 @@
+"""
+Clicksend platform for notify component.
+For more details about this platform, please refer to the documentation at
+https://clicksend.com/help
```

Comment:
Is this value fine: **https://home-assistant.io/docs/**
Because I haven't made a specific link for the documentation yet?